model/src/main/kotlin/config/LicenseFindingCuration.kt

```diff
@@ -77,11 +77,12 @@ data class LicenseFindingCuration(
     val comment: String = ""
 ) {
     init {
-        require(startLines.all { it >= 0 }) {
-            "All values for start lines must not be negative."
+        require(startLines.all { it > 0 }) {
+            "All start line values must not be positive."
```

Comment:
`it > 0`  means "must not be positive", this is a contradiction. Instead of saying "positive" why not say the easier to understand "greater than zero"

Reply:
Yeah, I just forgot to remove the "not". For myself, simple-word "positive" is easier to grasp than a rather clumsy "greater than zero"; but that's probably due to my mathematical background. Will change.